extensions/interactions/NumericInput/protractor.js

```diff
@@ -32,6 +32,8 @@ var expectInteractionDetailsToMatch = function(elem) {
 var submitAnswer = function(elem, answer) {
   elem.element(by.tagName('oppia-interactive-numeric-input')).
     element(by.tagName('input')).sendKeys(answer + '\n');
```

Comment:
We needed to get rid of it for the interactionHtml to work uniformly for all interactions.